### PR TITLE
Reduce multi-node tests to 10 pods, #30874

### PR DESCRIPTION
**Pull request by patriknw**
I missed this in https://github.com/akka/akka/pull/30873

References #30874

### CODE REVIEW

File: .github/workflows/multi-node.yml [hunk 1]
```diff
@@ -36,8 +36,8 @@ jobs:
 
       - name: Setup Pods
         run: |
-          # Stress tests are using 13 nodes.
-          ./kubernetes/setup.sh 15 multi-node-test.hosts tcp
+          # Start 10 pods. At most 10 MultiJvmNode (akka.cluster.StressSpec is currently disabled).
+          ./kubernetes/setup.sh 10 multi-node-test.hosts tcp
```
Comment (johanandren) [inline .github/workflows/multi-node.yml:40]: lol 

```
# we are using 13 nodes
$ use 15 nodes
```

**Review by johanandren — APPROVED** [review]

File: .github/workflows/multi-node.yml (hunk 1 above)
Comment (patriknw) [reply .github/workflows/multi-node.yml:40]: I guess the idea was 3 pods per node (we start 5 Kubernetes nodes)

Comment (johanandren) [reply .github/workflows/multi-node.yml:40]: Aha, nodes vs pods

**Comment by pvlugter** [comment]
👍🏼

**Pull request merged**